tests/store/test_status_samples_filters.py

```diff
@@ -0,0 +1,49 @@
+from alchy import Query
+
+from cg.constants.subject import Gender
+from cg.store.status_sample_filters import (
+    get_samples_with_loqusdb_id,
+    get_samples_without_loqusdb_id,
+)
+
+
+def test_get_samples_with_loqusdb_id(helpers, store, sample_store):
+    """Test sample extraction with Loqusdb ID."""
+
+    # GIVEN a sample observations that has been uploaded to Loqusdb
+    case = helpers.add_case(store)
+    sample = helpers.add_sample(store, loqusdb_id="uploaded_to_loqusdb")
```

Comment:
I think you should create a Mongo DB ID fixture (you needed it more then once 😄 )